docs/en/userinfo_endpoint.rst

```diff
@@ -20,15 +20,15 @@ The UserInfo Endpoint MUST only support the method HTTP GET :rfc:`2616` and MUST
 .. seealso::
 
  - https://openid.net/specs/openid-connect-core-1_0.html#UserInfo
- - https://openid.net/specs/openid-igov-openid-connect-1_0-03.html#rfc.section.4
+ - https://openid.net/specs/openid-igov-openid-connect-1_0-03.html#Section-4
 
 
 .. _userinfo_response:
 
 Response
 ++++++++
 
-The response of the UserInfo Endpoint MUST specify the value "application/jwt" in the "Content-Type".
+The response of the UserInfo Endpoint MUST specify the value "application/jose" in the "Content-Type".
```

Comment:
Non sono sicuro che il valore corretto è jose. Mi pareva di aver letto nelle specifiche che nel caso di nested jwt il content-type debba essere jwt. Verifichiamo.

Reply:
sarebbero equivalenti ma RFC7519 definisce il content type dei nested e registra in iana application/jwt, quindi mi torna ritornarlo a application/jwt, grazie

Reply:
https://github.com/italia/spid-cie-oidc-docs/pull/81/commits/c0a12c7add6d6b7c592f8e444fb770ed1667a0da